src/main/kotlin/no/nav/klage/oppgave/clients/KafkaOppgaveConsumer.kt

```diff
@@ -60,16 +56,13 @@ class KafkaOppgaveConsumer(
                             oppgaveService.updateHjemmel(oppgave.id, foundHjemler.first())
                         } else {
                             logger.debug("No need to store hjemmel")
+                            //send to oppgave-api
+                            oppgaveService.storeLocalCopy(oppgave)
```

Comment:
Jeg tror vi må lage lokal kopi langt oftere enn bare her. Jeg vil ha alle versjoner, egentlig uavhengig av hjemmel. Jeg tror tom vi må ha lagringen utenfor den første `if (oppgave.isKlage() && oppgave.isTildeltKlageenhet())` - vi ønsker å kunne se på historikken for klagene også mens de er hos førsteinstans.